### ISSUE TITLE
[BUG] Doesn't work

### ISSUE THREAD

**Issue by bartekpacia** [SwagLyrics/SwagLyrics-For-Spotify#2294]
**Describe the bug**
Was working fine. I updated Spotify app a few times, doesn't work anymore 

**To Reproduce**
1. Play any song on newest Spotify app on mac
2. `swaglyrics -c` 
3. Doesn't detect artist and song name (even Despacito doesn't work)

```
$ swaglyrics -c
Updating unsupported.txt from server.
Updated unsupported.txt successfully.
Lyrics unavailable for  by .


(Press Ctrl+C to quit)
^C
Sure boss, exiting.
```

**Expected behavior**
should work

**Desktop (please complete the following information):**
- MacOS 10.15.4
 - swaglyrics: 1.1.2
 - Spotify app version:  1.1.31.703.g256add22

**Comment by aadibajpai** [comment]
uh oh, this will be looked into very soon.

@MatejMecka my macOS friend can you reproduce this on your device?

Looks to be a problem with song detection

**Comment by MatejMecka** [comment]
On it

**Comment by MatejMecka** [comment]
Could not reproduce.

MacOS: 10.15.3
Spotify app version: 1.1.24.91.g4ca6d5eb

**Comment by MatejMecka** [comment]
Reproduced! The new update breaks the existing codebase

**Comment by aadibajpai** [comment]
> Reproduced! The new update breaks the existing codebase

So it worked with 10.15.3 but fails with 10.15.4?

**Comment by MatejMecka** [comment]
It worked with 1.1.24.91.g4ca6d5eb of Spotify but not 1.1.31.703.g256add22

**Comment by arcusdeer** [comment]
Same problem here, hit or miss with getting the actual song or 'Lyrics unavailable for  by .' I'm running Mojave 10.14.6 and Spotify 1.1.31.703.g256add22.

**Comment by aadibajpai** [comment]
Right, so that . happens when it can't seem to get the song info from spotify. I'll try to investigate what changed in this update and see what we can do about it later today. Hopefully it's a bug on their end because on macOS it just reads system info and it's not anything complicated like windows.

**Comment by aadibajpai** [comment]
I see other people have this problem too:

https://community.spotify.com/t5/Desktop-Mac/Applescript-no-longer-reports-Spotify-Connect-song-playing/td-p/4925257
https://community.spotify.com/t5/Desktop-Mac/You-broke-AppleScript-Again/m-p/4937134#M541534

**Comment by bartekpacia** [comment]
Looks like whole spotify plugin for `oh-my-zsh` broke, not SwagLyrics' fault

```
$ spotify status
Spotify is currently paused.
Artist:
Album: Californication
Track:
Position: 1:59 / 3:38
```

**Comment by MatejMecka** [comment]
This is the exact line causing the problem. When calling the method it returns nothing. Tested using Script Editor

https://github.com/SwagLyrics/SwSpotify/blob/master/SwSpotify/spotify.py#L96

**Comment by MatejMecka** [comment]
One way we can fix this is by parsing the spotify url which might be a PITA
